src/Tizen.NUI/src/public/Xaml/XamlParseException.cs

```diff
@@ -27,6 +27,23 @@ static private StringBuilder GetStackInfo()
             return ret;
         }
 
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public XamlParseException()
+        {
+        }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public XamlParseException( string message ) : base(message)
+        {
+           _unformattedMessage = message;
+        }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public XamlParseException( string message, Exception innerException = null ) : base(message, innerException)
```

Comment:
Ditto.

Reply:
I see.

Reply:
could you check the sites below for easy way of editing code?

- Visual Studio in Windows : https://stackoverflow.com/questions/1435691/visual-studio-format-entire-file

- VS Code in Ubuntu : https://stackoverflow.com/questions/29973357/how-do-you-format-code-in-visual-studio-code-vscode